production/helm/loki/templates/service.yaml

```diff
@@ -1,40 +1,26 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: {{ template "loki.fullname" . }}
-  namespace: {{ .Release.Namespace }}
+  name: {{ include "loki.fullname" . }}
   labels:
-    app: {{ template "loki.name" . }}
-    chart: {{ template "loki.chart" . }}
-    release: {{ .Release.Name }}
-    heritage: {{ .Release.Service }}
+    {{- include "loki.labels" . | nindent 4 }}
     {{- with .Values.service.labels }}
     {{- toYaml . | nindent 4 }}
     {{- end }}
   annotations:
     {{- toYaml .Values.service.annotations | nindent 4 }}
 spec:
   type: {{ .Values.service.type }}
-{{- if (and (eq .Values.service.type "ClusterIP") (not (empty .Values.service.clusterIP))) }}
-  clusterIP: {{ .Values.service.clusterIP }}
-{{- end }}
-{{- if .Values.service.loadBalancerSourceRanges }}
-  loadBalancerSourceRanges:
-  {{- range $cidr := .Values.service.loadBalancerSourceRanges }}
```

Comment:
That seems to be gone ?

Reply:
sorry. removed by accident. 
added it back.

Reply:
Did you added it back ?

Reply:
Oh ok you removed the clusterIP, I think it's fine.

Reply:
Why is it okay to remove a field that people could be using? This is a breaking change both in terms of client's expectations and in not being able to upgrade if they were using these fields. clusterIP is an immutable field so if it was set upgrades will fail. This should be added back or require a major version bump per semver.